web/styles/tooltips.css

```diff
@@ -35,6 +35,7 @@
 
         .tooltip-inner-content {
             line-height: 17px;
+            margin: 0;
```

Comment:
Another argument against `<p>` here is that we don't have to do this kind of pushback styling with `<div>` elements.